tests/fail/async-shared-mutable.tree.stderr

```diff
@@ -21,7 +21,7 @@ help: the accessed tag <TAG> later transitioned to Active due to a child write a
 LL |             *x = 1;
    |             ^^^^^^
    = help: this transition corresponds to the first write to a 2-phase borrowed mutable reference
-help: the accessed tag <TAG> later transitioned to Frozen due to a reborrow (acting as a foreign read access) at offsets [OFFSET]
+help: the accessed tag <TAG> later transitioned to Frozen due to a reborrow to create <TAG> (acting as a foreign read access) at offsets [OFFSET]
```

Comment:
Is it clear that the newly created tag is always the more interesting one here, rather than the tag of the pointer that is the input to the reborrow?

Reply:
Yeah, that's why the alternative solution to the linked issue is to make tag tracking also log reborrows. But I suspect that will be an unusable amount of information written to the terminal.